conf/modules/actuators_dummy.xml

```diff
@@ -4,9 +4,12 @@
   <doc>
     <description>
       Dummy actuators driver
+      This module prevents autoloading of actuators by fixedwing firmware
     </description>
   </doc>
   <header/>
-  <makefile/>
+  <makefile>
+    <configure name="ACTUATORS"/>
```

Comment:
Maybe we should rather use `<configure name="ACTUATORS" value="none"/>`?
That might be more clear and also consistent with the configure option we have for LEDs....